packages/orbit-components/src/ListChoice/index.js

```diff
@@ -78,17 +82,19 @@ StyledListChoiceContent.defaultProps = {
 };
 
 const ListChoice = (props: Props) => {
-  const { dataTest, icon, title, description, selectable, onClick, selected } = props;
+  const { dataTest, icon, title, description, selectable, onClick, selected, disabled } = props;
   const conditionalProps = {
     ...(selectable ? { "aria-checked": selected } : null),
   };
 
   return (
     <StyledListChoice
-      onClick={onClick}
+      onClick={!disabled ? onClick : null}
       data-test={dataTest}
-      onKeyDown={handleKeyDown(onClick)}
+      onKeyDown={!disabled ? handleKeyDown(onClick) : null}
```

Comment:
i suppose here should be undefined, not null :)

Reply:
It's the same, it won't do anything 🤷

Reply:
i know :) it's SC, in normal components we usually pass undefined in such cases, so i would follow same logic here, makes no difference in the end, but for consistency. Also i suppose it's better from perspective `null` and `undefined` differences, like here is more about something that hasn't been initialized

Reply:
That would mean that we have to go through the whole code base, we're not consistent when comes to usage of `null` or `undefined`.